app/src/main/java/tech/ula/utils/AndroidUtility.kt

```diff
@@ -398,4 +399,54 @@ class AcraWrapper {
     fun putCustomString(key: String, value: String) {
         ACRA.getErrorReporter().putCustomData(key, value)
     }
+}
+
+class UsageUtility(private val prefs: SharedPreferences) {
```

Comment:
I think this class name could be more descriptive, maybe something like `UserFeedbackUtility` or `UserFeedbackPreferences` to match other preference wrappers.

Reply:
I changed it to `UserFeedbackUtility` to match the other wrappers